pages/osx/xcode-select.md

```diff
@@ -0,0 +1,24 @@
+# xcode-select
+
+> Switch between different versions of Xcode and the included developer tools.
+> Also used to update the path to Xcode if it is moved after installation.
+
+- Install Xcode's command line tools:
+
+`xcode-select --install`
+
+- Select {{/path/to/Xcode.app/Contents/Developer}} as the active developer directory."
+
+`xcode-select -s {{/path/to/Xcode.app/Contents/Developer}`
+
+- Same as above, but the developer directory is automatically inferred:
```

Comment:
I'm unsure on how to phrase this better but I think we should explicitly say what the command does instead of forcing the user to read the previous example.

Reply:
Sure, alright. I just shortened it from what the man page says. It says this:

```man
As above, selects /Applications/Xcode.app/Contents/Developer
as the active developer directory. The Developer content directory
is automatically inferred by xcode-select.
```

Hows that?